src/_guides/openapi/specification/v3.1/_defaults.yml

```diff
@@ -1,2 +1,3 @@
-sidebar_version: "v3.1"
+sidebar_version: "3.1"
 sidebar_name: openapi_v3-1
+sidebar_title: OpenAPI Specification
```

Comment:
My bad, this can be removed, it's already present in `src/_guides/openapi/specification/_defaults.yml`